manifests/helm/templates/service.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "cerberus-mergeguard.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "cerberus-mergeguard.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ipFamilyPolicy: {{ .Values.service.ipFamilyPolicy }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)